src/Microsoft.AspNetCore.Razor.Runtime/project.json

```diff
@@ -56,9 +56,12 @@
     },
     "netstandard1.3": {
       "dependencies": {
+        "System.Collections.Concurrent": "4.0.12-*",
+        "System.Reflection": "4.1.0-*",
```

Comment:
The version of `System.Reflections.Extensions` we get is bringing in an older `System.Reflections`? Can we instead update the `System.Reflections.Extensions` version? If not, ick.